src/mmw/apps/modeling/tasks.py

```diff
@@ -210,6 +244,18 @@ def _get_precip_value(model_input):
         return None
 
 
+def convert_result_areas(pixel_width, results):
+    """
```

Comment:
I might have accidentally misinformed you.  Talking about the areas of geotiff pixels: the TR-55 module expects that number as input, not output.  For example:

> The cell_res parameter gives the resolution (size) of each cell. It is used for converting runoff, 
> evapotranspiration, and infiltration amounts from inches to volumes.

The part above were you calculate the pixel area can be used for this, I think.